unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendRequestHUD/IReceivedFriendRequestHUDView.cs

```diff
@@ -19,5 +19,6 @@ public interface IReceivedFriendRequestHUDView : IBaseComponentView<ReceivedFrie
         void SetState(LayoutState state);
         void Show(bool instant = false);
```

Comment:
this `Show` method is overriding one from the parent `IBaseComponentView` interface. You can just remove it from here